deploy-cluster-aws/fabfile.py

```diff
@@ -111,11 +113,22 @@ def install_rethinkdb():
         sudo('/etc/init.d/rethinkdb restart')
 
 
-# Install BigchainDB (from PyPI)
+# Install BigchainDB from PyPI
 @task
 @parallel
-def install_bigchaindb():
-    sudo('python3 -m pip install bigchaindb')
+def install_bigchaindb_from_pypi():
+    sudo('pip install bigchaindb')
```

Comment:
This should probably be `pip3 install bigchaindb`.
I think ubuntu already comes with pip for python2 installed by default so doing `pip install bigchaindb` will probably try to install bigchaindb with python2